src/org/ods/component/ScanWithSonarStage.groovy

```diff
@@ -149,13 +148,13 @@ class ScanWithSonarStage extends Stage {
         logger.debugClocked("${sonarProjectKey}-sq-scan", (null as String))
         retryComputeEngineStatusCheck(privateToken)
 
+        def targetReport = "sonarqube-report-${sonarProjectKey}.pdf"
+
         generateAndArchiveReports(
             sonarProjectKey,
-            context.buildTag,
-            sonarProperties['sonar.branch.name'].toString(),
-            context.sonarQubeEdition,
             !context.triggeredByOrchestrationPipeline,
```

Comment:
Question: By default SonarQube report is not archived when using the Orchestration pipeline.
Do we want to maintain this behavior?